newIDE/app/src/SceneEditor/index.js

```diff
@@ -915,6 +948,7 @@ export default class SceneEditor extends React.Component<Props, State> {
             ) => {
               return this._canObjectUseNewName(objectWithContext, newName);
             }}
+            onObjectCreated={this._addNewObjectInstance}
             onObjectSelected={this._onObjectSelected}
```

Comment:
For your interest, looking at this, I should have named this "onSelectObject" (because it's a callback that is called to have the parent do the selection, it's not a callback called when something is done).
Also, `_onObjectSelected` should be called `_selectObject`.